tools/orc/sgx.go

```diff
@@ -34,3 +41,106 @@ func sgxVerifySignature(bnd *bundle.Bundle) error {
 
```

Comment:
The `sgxVerifySignature` can probably now be removed as this is performed as part of the bundle validation?